lib/internal/url.js

```diff
@@ -874,13 +875,14 @@ URLSearchParams.prototype[util.inspect.custom] =
       output.push(`${innerInspect(list[i])} => ${innerInspect(list[i + 1])}`);
 
     const colorRe = /\u001b\[\d\d?m/g;
-    const length = output.reduce((prev, cur) => {
-      return prev + cur.replace(colorRe, '').length + ', '.length;
-    }, -', '.length);
+    const length = output.reduce(
+      (prev, cur) => prev + cur.replace(colorRe, '').length + separator.length,
+      -separator.length
+    );
```

Comment:
This code was originally from [lib/util.js](https://github.com/nodejs/node/blob/b5474c8cee47cda7ca0e3711bd39ebcd8a6d8964/lib/util.js#L830-L832), so you might be interested in changing the style there as well.